src/components/study-pane.js

```diff
@@ -777,6 +777,13 @@ const StudyPane = (props) => {
         }
     }
 
+    function scrollAndOpenVoltageLevel(vlId) {
+        if (!network) return;
+        setDisplayedVoltageLevelId(vlId);
```

Comment:
don't call this, showVOltageLevelDiagram will update the url and this will trigger an useEffect to update displayedVoltageLevelId . (the same bug is present in onClickNmKConstraint, we should clean it later)